tests/unit/neptune/new/internal/backends/test_hosted_client.py

```diff
@@ -115,6 +116,29 @@ def test_project_listing(self, swagger_client_factory):
         # then:
         self.assertEqual(["org1/project1", "org2/project2"], returned_projects)
 
+    def test_invite_to_workspace(self, swagger_client_factory):
+        swagger_client = self._get_swagger_client_mock(swagger_client_factory)
+
+        invite_to_workspace(
+            username="tester1",
+            workspace="org2",
+            api_token=API_TOKEN,
+        )
+
+        self.assertEqual(swagger_client.api.createOrganizationInvitations.call_count, 1)
```

Comment:
Let's cleanup a little bit here. Create a structure like `given`/`when`/`then` (could be separated with comments as in case of other tests). Instead of `call_args.kwargs` you should be able to call something like `called_once_with(...)` or something like that.